Reservia_files/style.css

```diff
@@ -0,0 +1,620 @@
+    border-radius: 2%;
+    margin-left: 2%;
+    margin-right: 2%;
+    margin-bottom: 2%;
+   
+    
+}
+/*.section1>#box1>.imgbox1{
+    width: 91%;
+    height: 60%;
+    padding-top: 3%;
+    padding-left: 4%;
+    Padding-right:3%;
+    border-top-left-radius: 7%;
+    border-top-right-radius:7% ;
+}*/
+
+ /*#star{
+    width: 10%;
+    height: 10%; 
+    padding-top:0%;
+ }*/
+
+ /*h5{
+     font-size: large;
+     text-align: left;
+     padding-left: 0px;
+ }
+ p{
+    text-align: left;
+    padding-left: 0px;
+    padding-top: 0px;
+ }
+
+ span{
+     font-weight: bold;
+ }
+*/
+ /*.section1>#box2>img{
+    width: 10%;
+    height: 10%;
+    padding-top: 2%;
+    padding-left: 2%;
+    Padding-right:0%;
+    border-top-left-radius: 7%;
+    border-bottom-left-radius: 7%;
+    
+    
+
+ }*/
+ /*.imgbox1{
+   
+    width: 97%;
+    height: 50%;
+    padding-top: 3%;
+    padding-left: 4%;
+    Padding-right:3%;
+    border-top-left-radius: 7%;
+    border-top-right-radius:7% ;
+ }*/
+ 
+
+ /*#hostimg7, #hostimg8,#hostimg9{
+    width: 50%;
+    height: 100%;
+    border-top-left-radius: 7%;
+    border-bottom-left-radius: 7%;
+    border-bottom-right-radius: 0%;
+    border-top-right-radius: 0%;
+ }*/
+
+
+/*.imgbox1{
+    width: 97%;
+    height: 50%;
+    padding-top: 3%;
+    padding-left: 4%;
+    Padding-right:3%;
+    border-top-left-radius: 7%;
+    border-top-right-radius:7% ;
+}*/
+/*#hostimg1{
+    width: 92%;
+    height: 50%;
+    padding-top: 3%;
+    padding-left: 4%;
+    Padding-right:4%;
+    border-top-left-radius: 7%;
+    border-top-right-radius:7% ;
+}
+#hostimg2,#hostimg3,#hostimg4,#hostimg5,#hostimg6{
+    width: 97px;
+    height: 50px;
+    padding-top: 0px;
+    padding-left: 0px;
+    Padding-right:3px;
+    border-top-left-radius: 7px;
+    border-top-right-radius:7px ;
+}
+*/
+.imgbox1{
+   
+    width: 94%;
+    height: 50%;
+    padding-top: 3%;
+    padding-left: 3%;
+    Padding-right:3%;
+    border-top-left-radius: 7%;
+    border-top-right-radius:7% ;
+   /* padding-top: 0px;
+    padding-left: 0px;
+    Padding-right:3px;
+   border-top-left-radius: 7px;
+    border-top-right-radius:7px ;*/
+}
+
+.imgbox2{
+    width: 50%;
+    height: 90%;
+    padding-top: 3%;
+    padding-left: 3%;
+    padding-bottom:3%;
+    border-top-left-radius: 10%;
+  
+    border-bottom-left-radius: 10%;
+   
+    
+    
+    
+}
+
+.figcaption1{
+    padding-left: 3%;
+    font-weight: bold;
+
+}
+.figcaption2{
+    
+    font-weight: bold;
+    float:right;
+    padding-top:3%;
+   
+    padding-right: 3%;
+    
+    
+
+}
+#idFigcaption1{
+    padding-right: 11%; 
+}
+#idFigcaption2{
+    
+    font-size:0.83em;
+    padding-right:8%;
+
+}
+
+p{
+    text-align: left;
+    padding-left: 3%;
+    padding-top: 0px;
+ }
+
+ span{
+     font-weight: bold;
+ }
+
+  #star{
+    padding-left: 1%;
+    width:30%;
+    height: 20%; 
+    padding-top:0%;
+ }
+ .section2{
+    display: grid;
+    background-color:white;
+    
```

Comment:
empty lines for nothing. to be removed

useless code commented, it should be removed instead.

Reply:
Empty lines have been removed.